packages/babel-traverse/test/fixtures/conversion/implicit-var-arguments/exec.js

```diff
@@ -0,0 +1,11 @@
+var arguments = [1, 2, 3];
+var arr = () => arguments[0];
+
+arr(); // 1
```

Comment:
Can you add an explicit `expect` here rather than just a comment?

Reply:
Yeah, I can do that.